src/tests/ftest/util/telemetry_utils.py

```diff
@@ -0,0 +1,139 @@
+        """Obtain the specified metric informaton for each host.
+
+        Args:
+            name (str): Comma-separated list of metric names to query.
+
+        Returns:
+            dict: a dictionary of host keys linked to metric data for each
+                metric name specified
+
+        """
+        info = {}
+        self.log.info("Querying telemetry metric %s from %s", name, self.hosts)
+        for host in self.hosts:
+            data = self.dmg.telemetry_metrics_query(host=host, metrics=name)
+            info[host] = {}
+            if "response" in data:
+                if "metric_sets" in data["response"]:
+                    for entry in data["response"]["metric_sets"]:
+                        info[host][entry["name"]] = {
+                            "description": entry["description"],
+                            "metrics": entry["metrics"]
+                        }
+        return info
+
+    def get_container_metrics(self):
+        """Get the container telemetry metrics.
+
+        Returns:
+            dict: dictionary of  of errors detected
+
+        """
+        names = [
+            "engine_container_ops_open_total",
+            "engine_container_ops_open_active",
+            "engine_container_ops_close_total",
+            "engine_container_ops_destroy_total",
+        ]
+        data = {}
+        info = self.get_metrics(",".join(names))
+        self.log.info("Container Telemetry Information")
+        for host in info:
+            data[host] = {key: 0 for key in names}
+            for name in names:
+                if name in info[host]:
+                    for metric in info[host][name]["metrics"]:
+                        self.log.info(
+                            "  %s (%s): %s (%s)",
+                            info[host][name]["description"], name,
+                            metric["value"], host)
+                        data[host][name] = metric["value"]
+        return data
+
+    def check_container_metrics(self, open_count=None, active_count=None,
+                                close_count=None, destroy_count=None):
+        """Verify the container telemetry metrics.
+
+        Args:
+            open_count (int, optional): Number of times cont_open has been
+                called. Defaults to None.
+            active_count (int, optional): Number of open container handles.
+                Defaults to None.
+            close_count (int, optional): Number of times cont_close has been
+                called. Defaults to None.
+            destroy_count (int, optional): Number of times cont_destroy has been
+                called. Defaults to None.
+
+        Returns:
+            list: list of errors detected
+
+        """
+        errors = []
+        names = [
+            "engine_container_ops_open_total",
+            "engine_container_ops_open_active",
+            "engine_container_ops_close_total",
+            "engine_container_ops_destroy_total",
+        ]
```

Comment:
Could be value in defining this list externally -- as far as I'm aware there aren't plans for a lot more container metrics in the future, but if we do add something, it would be better to avoid having to add it to multiple places in the tests.